libkbfs/interfaces.go

```diff
@@ -506,11 +506,7 @@ type encryptionKeyGetter interface {
 		kbfscrypto.TLFCryptKey, error)
 }
 
-// KeyManager fetches and constructs the keys needed for KBFS file
-// operations.
-type KeyManager interface {
-	encryptionKeyGetter
-
+type mdDecryptionKeyGetter interface {
```

Comment:
Could combine this with `encryptionKeyGetter` to have a single `keyGetter` interface. I'm not sure which way is better.